### DISCUSSION TITLE
Wallpaper not being set

### DISCUSSION THREAD

**Discussion by dsriggs1** [nix-community/stylix#411] [Q&A] [answered]
In my configuration file if I include the following:
 stylix.image = /wallpaper.jpg;

This sets a wallpaper and color scheme for my login menu, but when I add the same line to my home.nix file the wallpaper is not being set on my desktop, but the color scheme is being applied to my apps (so it seems the image is being picked up). I can't figure how to get the desktop image to appear though.

Previously I was using pywal to set the wallpaper and color scheme and I thought this might be interfering with stylix, but removing pywal from my program list hasn't fixed the issue (it just results in a black screen where the wallpaper would be).

I am using qtile as the WM if that is relevant.

Thanks for any suggestions on what might be going wrong here.

**Comment by danth** [comment, marked as answer]
We don't have a module for qtile yet, so that's why the wallpaper isn't showing up.

**Reply by dsriggs1** [reply]
Ah. Thanks for the reply that makes sense. Are there any plans to add a module for qtile in the near future?

**Reply by danth** [reply]
Not currently; feel free to open an issue about it though.

**Comment by lifeashansen** [comment]
> In my configuration file if I include the following: stylix.image = /wallpaper.jpg;
> 
> This sets a wallpaper and color scheme for my login menu, but when I add the same line to my home.nix file the wallpaper is not being set on my desktop, but the color scheme is being applied to my apps (so it seems the image is being picked up). I can't figure how to get the desktop image to appear though.
> 
> Previously I was using pywal to set the wallpaper and color scheme and I thought this might be interfering with stylix, but removing pywal from my program list hasn't fixed the issue (it just results in a black screen where the wallpaper would be).
> 
> I am using qtile as the WM if that is relevant.
> 
> Thanks for any suggestions on what might be going wrong here.

I have the same issue but on hyprland with nixos though

**Reply by danth** [reply]
AFAIK Hyprland wallpapers are only set with Home Manager
